modules/ols-enabling-cluster-interaction.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-cluster-interaction_{context}"]
+= Enabling cluster interaction
+
+Modify the `OLSConfig` custom resource to enable the cluster interaction feature.
+
+:FeatureName: The cluster interaction feature
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped CR.
+
+* You have configured the large language model (LLM) provider.
+
+* You have installed the {ols-long} Operator.
+
+.Procedure 
+
+. In the {ocp-product-title} web console, click *Operators* -> *Installed Operators*. 
+
+. Click {ols-long} Operator.
+
+. Click *OLSConfig*, then click the `cluster` configuration instance in the list.
+
+. Click the *YAML* tab.
+
+. Set the `spec.ols.introspectionEnabled` parameter to `true` to enable cluster interaction: 
++
+.Example `OLSconfig` CR file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+spec:
+  ols:
+    introspectionEnabled: true  
+----
+
+. Click *Save*.
+
+. Access the {ols-long} virtual assistant and submit a question associated with the custom content that was added to the LLM.
```

Comment:
Won't stop merge review over this, but maybe this could be more of a verification step rather than the last step of the actual procedure? Up to you though